firebase-installations/customer-lint-checks/src/main/java/com/google/firebase/installations/lint/IncompatibleIidVersionDetector.kt

```diff
@@ -59,15 +53,29 @@ class IncompatibleIidVersionDetector : Detector() {
                     if (isIncompatibleVersion(coordinates)) {
                         context.report(INCOMPATIBLE_IID_VERSION,
                                 Location.create(context.file),
-                                "Incompatible IID version found in variant ${variant.name}: ${lib.name}")
+                                "Incompatible IID version found in variant ${variant.name}: ${lib.name.removeSuffix("@aar")}.\n" +
+                                        "To resolve this issue, consider updating the build file to the latest firebase-iid version.")
                     }
                 }
             }
         }
     }
 
     private fun isIncompatibleVersion(coordinates: MavenCoordinates): Boolean {
-        // TODO: implement version check
-        return coordinates.version == "0.0.0"
+        val versionComponents = coordinates.version.split('.').toTypedArray()
```

Comment:
Can you tell split to split this into maximum 3 components?
If yes, please do so.

Reply:
Done.